app/api/search/specs/search.spec.js

```diff
@@ -578,12 +579,24 @@ describe('search', () => {
     });
 
     describe('allAggregations', () => {
-      it('should return all aggregations', async () => {
+      it('should return all aggregations when clasification is not enabled', async () => {
         userFactory.mock(undefined);
         const allAggregations = await search.search({ allAggregations: true }, 'en');
         const aggregationsIncluded = Object.keys(allAggregations.aggregations.all);
         expect(aggregationsIncluded).toMatchSnapshot();
       });
+      it('should return all aggregations when clasification is enabled', async () => {
```

Comment:
Again, wrong spelling in classification.

Reply:
Approach and tests changed.